crates/solidity/inputs/language/src/dsl.rs

```diff
@@ -546,7 +546,7 @@ slang_grammar! {
 
         parser UsingDirective = ((UsingKeyword (UsingDirectivePath | UsingDirectiveDeconstruction) ForKeyword (Asterisk | TypeName) ({ introduced in "0.8.13" GlobalKeyword } ?)) terminated by Semicolon) ;
```

Comment:
do we need to version the callsite in `UsingDirective` as well?

Reply:
It's not needed and I think it's clearer this way, since it has only one use site